backend/cmd/plugins.go

```diff
@@ -155,15 +155,15 @@ func addPluginRoutes(config *HeadlampConfig, r *mux.Router) {
 		pluginListURLs = make([]string, 0)
 	}
 
-	r.HandleFunc("/plugins/list", func(w http.ResponseWriter, r *http.Request) {
+	r.HandleFunc("/plugins/paths", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think `paths` represent a logical concept of how we organize plugins that's not necessarily interesting for what we're trying to do.
The frontend needs a list of the plugins available. If we want to not use the old path (probably a good idea), then just `/plugins` probably works?

Reply:
It's changed to /plugins now.